package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "devicon",
-  "version": "2.15.0",
+  "version": "2.15.1",
```

Comment:
Please reverse all changes to this file. Only `devicon.json` and the icon files should be edited/added